joladnijo/settings.py

```diff
@@ -76,9 +77,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'joladnijo',
-        'USER': 'joladnijo',
+        'ENGINE': os.environ.get('DBENGINE', ''),
+        'NAME': os.environ.get('DBNAME', ''),
```

Comment:
Szerintem ez sem kell, hogy valtozzon kulonbozo kornyezetekben.
```suggestion
        'NAME': 'joladnijo',
```